rmw_fastrtps_cpp/src/rmw_publisher.cpp

```diff
@@ -72,6 +75,16 @@ rmw_create_publisher(
     node->implementation_identifier,
     eprosima_fastrtps_identifier,
     return nullptr);
+  RMW_CHECK_ARGUMENT_FOR_NULL(qos_policies, nullptr);
```

Comment:
If I understand correctly, this is going to exit the function in `qos_policies` is a nullptr. Is this not a case supported by the API? I can not find documentation for it in https://docs.ros2.org/foxy/api/rmw/rmw_8h.html#a149b0600fa9ce696c83a63f0e1e806ce

Reply:
Make sure to look at the latest version: https://docs.ros2.org/galactic/api/rmw/rmw_8h.html#ac74022d90eb22d95b082d4808670f8a9

It has better docs 👍

Reply:
aaaaaah, thanks! Tried rolling and fallback to foxy but completely forgot about Galactic.